Scripts/Core/EditorVR.Interfaces.cs

```diff
@@ -282,6 +293,27 @@ internal void ConnectInterfaces(object obj, Transform rayOrigin = null)
 				if (requestStencilRef != null)
 					requestStencilRef.requestStencilRef = RequestStencilRef;
 
+				var linkedTool = obj as ILinkedTool;
+				if (linkedTool != null)
+				{
+					List<ILinkedTool> otherTools;
+					var type = obj.GetType();
+					if (m_LinkedTools.TryGetValue(type, out otherTools))
+					{
+						foreach (var otherTool in otherTools)
+						{
+							otherTool.otherTools.Add(linkedTool);
+						}
+
+						linkedTool.otherTools.AddRange(otherTools);
+					}
+					else
+					{
+						linkedTool.primary = true;
```

Comment:
I'm not clear on what makes a tool primary. My preference would be to remove this distinction.